### DISCUSSION TITLE
Public key certificate

### DISCUSSION THREAD

**Discussion by BuiltByMarcelo** [fuzzzerd/fmdata#212] [Q&A]
Hi!
We are using FMData (.NET 5 web application )to connect to a FileMaker instance that is secured by an SSL certificate.
I am wondering how can I include the public key certificate (CER) to validate the connection?
We are hosting the app in Azure AppService and currently getting a System.Security.Authentication.AuthenticationException: The remote certificate is invalid because of errors in the certificate chain: UntrustedRoot error message.
Thanks!

**Comment by fuzzzerd** [comment]
FMData uses the `HttpClient` that you provide via dependency injection, so you can customize that by using `IHttpClientFactory`: https://docs.microsoft.com/en-us/dotnet/architecture/microservices/implement-resilient-applications/use-httpclientfactory-to-implement-resilient-http-requests

I'd be curious if you are able to make any kind of connection, possibly with a console app running locally, just to see if you can make any kind of HTTP connections to your FileMaker server using standard .NET APIs?
